package.json

```diff
@@ -7,7 +7,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "webpack-dev-server --progress --colors",
     "watch": "NODE_ENV=development webpack -d -w",
```

Comment:
I get an error when I run this line, about the `devtool` syntax being invalid. Looks like that's because `-d` no longer stands for `development` in webpack 5, but instead stands for `devtool`. The way I got it to work was to change this line to:

```bash
webpack --mode=development --watch
```

Note that setting it to development mode also sets `NODE_ENV` to `development` ([source](https://webpack.js.org/configuration/mode#usage)).

Gosh, once we remove the bulk of the JS, let's consider moving to one of the simpler alternatives to webpack!